src/main/java/org/jarvis4ops/controller/SlachController.java

```diff
@@ -205,5 +207,35 @@ public String postMaxWipBreachedSlack(@RequestBody String jiraMaxWipCountRequest
 		slackHelper.postOnSlack(gson.toJson(slachAttachments), "jiraBots");
 		return "200";
 	}
+	
+	@RequestMapping(value = "/postInvalidTimeOnSlack", method = { RequestMethod.POST })
+	public String slackMessageForInvalidTime(@RequestBody String invalidJiraTimeIssues) {
+		
+		Gson gson = new Gson();
+		Type newType = new TypeToken<HashMap<String, TimeBean>>(){}.getType();
+		HashMap<String, TimeBean> map = new Gson().fromJson(invalidJiraTimeIssues, newType);
+		
+		slackHelper.getInvlaidTimes(map);
+		SlachBean slachBean = new SlachBean();
+		
+		slachBean.setFallback("@here: " + map.toString() + " " + configObj.getGreatJobTitleMsg() + configObj.getIncidentsResolvedMsg());
+
+		slachBean.setText(configObj.getInvalidTimeMessage());
+		slachBean.setTitle(map.toString() + " " + configObj.getGreatJobTitleMsg());
+		SlachAttachments slachAttachments = new SlachAttachments();
+		List<SlachBean> slachBeanList = new ArrayList<SlachBean>(1);
+		slachBeanList.add(slachBean);
+		slachAttachments.setAttachments(slachBeanList);
+		
+		//log.info("Json Value: ", gson.toJson(attachments));
+		log.info("Json Value: " + gson.toJson(slachAttachments));
+		
+		for (String jiraItem: map.keySet())
+		{
+			slackHelper.postOnSlackChat(gson.toJson(slachAttachments), "@"+ map.get(jiraItem).getSlackId(), "JIRA ID # " + jiraItem + " is closed in less than 5 minutes. Please follow the Kanban guidelines to resolve issues");
```

Comment:
Use a constant.
Also make "5" as configurable using SlackMessagingConstants and props file.